managed/models/artifact_model.go

```diff
@@ -51,13 +52,15 @@ type BackupStatus string
 
 // BackupStatus status (in the same order as in artifacts.proto).
 const (
-	PendingBackupStatus        BackupStatus = "pending"
-	InProgressBackupStatus     BackupStatus = "in_progress"
-	PausedBackupStatus         BackupStatus = "paused"
-	SuccessBackupStatus        BackupStatus = "success"
-	ErrorBackupStatus          BackupStatus = "error"
-	DeletingBackupStatus       BackupStatus = "deleting"
-	FailedToDeleteBackupStatus BackupStatus = "failed_to_delete"
+	PendingBackupStatus             BackupStatus = "pending"
+	InProgressBackupStatus          BackupStatus = "in_progress"
+	PausedBackupStatus              BackupStatus = "paused"
+	SuccessBackupStatus             BackupStatus = "success"
+	ErrorBackupStatus               BackupStatus = "error"
+	DeletingBackupStatus            BackupStatus = "deleting"
+	FailedToDeleteBackupStatus      BackupStatus = "failed_to_delete"
```

Comment:
I wonder if we can rename this to `delete_failed` as well?

Reply:
You mean PMM API? We can rename it there as well

Reply:
TBH I don't think it's a good idea to do general refactoring in the scope of such massive task. Changes you're suggesting require changes in several files and in database. Let's do it in scope of other task.

Reply:
Ah, that's old code. Ok then.